system/ee/ExpressionEngine/Addons/search/upd.search.php

```diff
@@ -149,6 +149,10 @@ public function update($current = '')
                 ->update('actions', array('csrf_exempt' => 1));
         }
 
+        if (version_compare($current, '2.3', '<')) {
+            ee()->smartforge->add_key('search', 'no_result_page');
```

Comment:
This column does not exist, so key can't be added

Reply:
@intoeetive , how could I add a new column to the database?

Reply:
You would need to use `ee()->smartforge->add_column`
But is there a need for this column at all? Could it not be always fetched from tag parameter instead?

Reply:
Thank you, @intoeetive . Yeah, it's need. The values of tag parameters are present only on the form tag, not on the results one.